build.gradle

```diff
@@ -269,3 +280,5 @@ idea.project.ipr {
 apply from: "${rootDir}/scripts/jacoco.gradle"
 
 apply from: publishPlugin
+
```

Comment:
Please avoid extra empty lines. One at the end of each file (as the `EOF` symbol) is fairly enough.